### PR TITLE
Aggregate errors with @inaccessible

### PR DESCRIPTION
**Pull request by benweatherman**
Previously, `removeInaccessibleElements` would throw an error the first time it encountered an inaccessible mismatch for types & fields.

With this change, the code will continue to parse the schema and report an `ErrGraphQLValidationFailed` that contains all the mismatches.

### CODE REVIEW

File: internals-js/src/inaccessibleSpec.ts [hunk 1]
```diff
@@ -85,4 +87,8 @@ export function removeInaccessibleElements(schema: Schema) {
       toRemove.forEach(f => f.remove());
     }
   }
+
+  if (errors.length) {
+    throw ErrGraphQLValidationFailed(errors)
```
[the remark this answers is not in the document]
Comment (benweatherman) [reply internals-js/src/inaccessibleSpec.ts]: Good idea! Added in https://github.com/apollographql/federation/pull/1563/commits/f533823fbd37a20c5fa9f34e39e3b1dbb0cc9372

File: internals-js/src/inaccessibleSpec.ts [hunk 2]
```diff
@@ -92,4 +94,8 @@ export function removeInaccessibleElements(schema: Schema) {
       toRemove.forEach(f => f.remove());
     }
   }
+
+  if (errors.length > 0) {
+    throw ErrGraphQLValidationFailed(errors, 'Schema has invalid usages of the the @inaccessible directive.');
```
Comment (pcmanus) [inline internals-js/src/inaccessibleSpec.ts:99]: nit: s/the the/the/

**Review by pcmanus — APPROVED** [review]

File: internals-js/src/inaccessibleSpec.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (benweatherman) [reply internals-js/src/inaccessibleSpec.ts]: Updated, then updated again with my typo removed

**Comment by benweatherman** [comment]
> wonder if we should try to harmonize those part a bit

At first blush, I think it's OK to throw multiple types of errors from `toAPISchema`. Sometimes they'll be an aggregated error, sometimes they'll be something else. But happy to talk through this more, I'm sure there's a lot there I'm not thinking about.

Barring that, I'd say these changes are ready to merge.

**Comment by pcmanus** [comment]
> At first blush, I think it's OK to throw multiple types of errors from `toAPISchema`

Yeah, you're right, that's fine.

> I'd say these changes are ready to merge.

+1

**Pull request merged**